### PR TITLE
Option to overwrite default max row numbers

### PR DESCRIPTION
**Pull request by Bermpje**
We have a use case where specific users are allowed to retrieve a larger amount of rows than allowed by default. To so so, we created an extra maxrows_override field for the following drivers:

- athena
- mysql
- mysql2
- postgresql
- sqlserver
- snowflake
- unixodbc

### CODE REVIEW

File: server/drivers/athena/index.js [hunk 1]
```diff
@@ -268,8 +276,29 @@ const fields = [
...
+ * @returns
+ */
+function resolvePositiveNumber(num, defaultValue) {
+  if (num == null) return defaultValue;
+  if (typeof num === 'string') num = Number.parseInt(num, 10);
+  if (typeof num !== 'number') return defaultValue;
+  if (!Number.isFinite(num)) return defaultValue;
+  if (num > 0) return num;
+  return defaultValue;
+}
```
Comment (rickbergfalk) [inline server/drivers/athena/index.js:300]: Could this be extracted and put in util/lib directory with some tests?

**Review by rickbergfalk — CHANGES_REQUESTED** [review]
Thanks for contribution @Bermpje!

Could the `resolvePositiveNumber` be moved to its own file in `server/lib` and have a few test cases written for it?

File: server/drivers/athena/index.js (hunk 1 above)
Comment (Bermpje) [reply server/drivers/athena/index.js]: Done!

**Review by rickbergfalk — APPROVED** [review]
Looks great! Thanks @Bermpje!

**Pull request merged**